online-shop/src/app/product-list/product-list.component.scss

```diff
@@ -0,0 +1,61 @@
+.main-container{
+    background-color: rgb(133, 173, 173,0.3);
+    margin:0;
+    padding: 0;
+    width: 100%;
+    height: auto;
+
+}
+
+.toolbar{
+    padding: 20px;
+    width: 100%;;
+    display: inline-block;
+    height: auto;
+    background-color: rgb(133, 173, 173,0.6);
+    h1{
+        float: left;
+    }
+}
+
+button{
+    border-radius: 5px;
+    border:0;
+    align-items: center;
+    background-color: gray;
+    text-transform: uppercase;
+    margin-top: 10px;
+    margin-right: 100px;
+    width: 100px;
+    height: 50px;
+    float:right;
+    color: white;
+    transition: 0.4s;
+    font-weight: 800;
+}
```

Comment:
duplicated CSS in multiple components. to solve this, one option is to add global styles in styles.scss file
The second option is to create separate style classes, import them in styles.scss file, and use them in HTML templates

the second option it's easy to scale and maintain